README.md

```diff
@@ -143,7 +143,7 @@ To enable Cognito authentication on the Atlantis ALB, specify the following argu
 
 ```hcl
 alb_authenticate_cognito = {
-  user_pool_arn       = "arn:aws:cognito-idp:us-west-2:1234567890:userpool/us-west-2_aBcDeFG"
+  user_pool_arn       = "arn:${data.aws_partition.current.partition}:cognito-idp:us-west-2:1234567890:userpool/us-west-2_aBcDeFG"
```

Comment:
There is no need to edit examples in README. Please revert all these changes in README.